common/federation/src/main/java/app/coronawarn/server/common/federation/client/FederationGatewayClient.java

```diff
@@ -55,4 +55,23 @@
       byte[] raw,
       @RequestHeader("batchTag") String batchTag,
       @RequestHeader("batchSignature") String batchSignature);
+
```

Comment:
unnecessary blank line

Reply:
After removing the line codeql is failing so I reverted it.